lib/smart_proxy_dynflow/io_buffer.rb

```diff
@@ -71,7 +71,7 @@ def close
       def read_available!
         data = ''
         loop { data += @io.read_nonblock(4096) }
-      rescue IO::WaitReadable # rubocop:disable Lint/HandleExceptions
+      rescue IO::WaitReadable
```

Comment:
```suggestion
      rescue IO::WaitWritable # rubocop:disable Lint/SupressedException
```

Reply:
Updated this changes here https://github.com/theforeman/smart_proxy_dynflow/pull/118